EXILED/Exiled.Events/Patches/Events/Player/EscapingAndEscaped.cs

```diff
@@ -98,9 +99,28 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                 {
                     // GrantAllTickets(ev)
                     new CodeInstruction(OpCodes.Ldloc, ev.LocalIndex).WithLabels(labels),
-                    new(OpCodes.Call, Method(typeof(Escaping), nameof(GrantAllTickets))),
+                    new(OpCodes.Call, Method(typeof(EscapingAndEscaped), nameof(GrantAllTickets))),
                 });
 
+            newInstructions.InsertRange(newInstructions.Count - 1, new CodeInstruction[]
+            {
+                // loading EscapingEventArgs instance 2 times
+                new(OpCodes.Ldloc_S, ev.LocalIndex),
+                new(OpCodes.Ldloc_S, ev.LocalIndex),
+
+                // ev.Player
+                new(OpCodes.Callvirt, PropertyGetter(typeof(EscapingEventArgs), nameof(EscapingEventArgs.Player))),
+
+                // ev.EscapeScenario
+                new(OpCodes.Callvirt, PropertyGetter(typeof(EscapingEventArgs), nameof(EscapingEventArgs.EscapeScenario))),
```

Comment:
shouldnt it be
```
ldloc.s ev
callvirt EscapingEventArgs::Player
ldloc.s ev
callvirt EscapingEventArgs::EscapeScenario
```
?
because that patch tries to get `EscapingEventArgs::EscapeScenario` for player object